From e1d3158523903ce2f82abc815898cb7e8fd3255a Mon Sep 17 00:00:00 2001
From: Stefan Lange <s.lange@gateware.de>
Date: Tue, 29 Nov 2016 11:52:50 +0100
Subject: [PATCH] TQMT104x device trees: set EC2 RGMII phy to output 125MHz on
 CLK_OUT pin

Program RGMII phy DP83867 (address 0x05) at MAC EC2 to output 125MHz
on CLK_OUT pin.
This pin is wired to provide the GTX clock to MAC EC2.

Signed-off-by: Stefan Lange <s.lange@gateware.de>
---
 arch/powerpc/boot/dts/fsl/tqmt1040.dts | 2 ++
 arch/powerpc/boot/dts/fsl/tqmt1042.dts | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/powerpc/boot/dts/fsl/tqmt1040.dts b/arch/powerpc/boot/dts/fsl/tqmt1040.dts
index 5ac3d48..a2b3227 100644
--- a/arch/powerpc/boot/dts/fsl/tqmt1040.dts
+++ b/arch/powerpc/boot/dts/fsl/tqmt1040.dts
@@ -279,12 +279,14 @@
 					ti,tx-internal-delay = <DP83867_RGMIIDCTL_2_50_NS>;
 					ti,rx-internal-delay = <DP83867_RGMIIDCTL_2_00_NS>;
 					ti,fifo-depth = <DP83867_PHYCR_FIFO_DEPTH_4_B_NIB>;
+					ti,dp83867-clk-out-sel = <DP83867_IOMUXCFG_CLK_O_SEL_REF_CLK>;
 				};
 				phy_rgmii_1: ethernet-phy@05 {
 					reg = <0x05>;
 					ti,tx-internal-delay = <DP83867_RGMIIDCTL_2_50_NS>;
 					ti,rx-internal-delay = <DP83867_RGMIIDCTL_2_00_NS>;
 					ti,fifo-depth = <DP83867_PHYCR_FIFO_DEPTH_4_B_NIB>;
+					ti,dp83867-clk-out-sel = <DP83867_IOMUXCFG_CLK_O_SEL_CHA_RX>;
 				};
 				phy_sgmii_1: ethernet-phy@03 {
 					reg = <0x03>;
diff --git a/arch/powerpc/boot/dts/fsl/tqmt1042.dts b/arch/powerpc/boot/dts/fsl/tqmt1042.dts
index 601083d..7b678d2 100644
--- a/arch/powerpc/boot/dts/fsl/tqmt1042.dts
+++ b/arch/powerpc/boot/dts/fsl/tqmt1042.dts
@@ -284,12 +284,14 @@
 					ti,tx-internal-delay = <DP83867_RGMIIDCTL_2_50_NS>;
 					ti,rx-internal-delay = <DP83867_RGMIIDCTL_2_00_NS>;
 					ti,fifo-depth = <DP83867_PHYCR_FIFO_DEPTH_4_B_NIB>;
+					ti,dp83867-clk-out-sel = <DP83867_IOMUXCFG_CLK_O_SEL_REF_CLK>;
 				};
 				phy_rgmii_1: ethernet-phy@05 {
 					reg = <0x05>;
 					ti,tx-internal-delay = <DP83867_RGMIIDCTL_2_50_NS>;
 					ti,rx-internal-delay = <DP83867_RGMIIDCTL_2_00_NS>;
 					ti,fifo-depth = <DP83867_PHYCR_FIFO_DEPTH_4_B_NIB>;
+					ti,dp83867-clk-out-sel = <DP83867_IOMUXCFG_CLK_O_SEL_CHA_RX>;
 				};
 				phy_sgmii_0: ethernet-phy@1D {
 					reg = <0x1D>;
-- 
1.9.1

